.github/workflows/pre-commit.yml

```diff
@@ -9,12 +9,6 @@ jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: actions/setup-python@v1
-    - name: set PY
-      run: echo "::set-env name=PY::$(python --version --version | sha256sum | cut -d' ' -f1)"
```

Comment:
@callumrollo this should fix the failing GitHub Action in your PR. I'll fix Travis-CI in another one b/c I also want to move it away from Travis-CI.